autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/management/sorting/CredentialListSorterByTitleAndDomain.kt

```diff
@@ -30,7 +30,7 @@ interface CredentialListSorter {
     fun comparator(): Collator
 }
 
-@ContributesBinding(FragmentScope::class)
+@ContributesBinding(AppScope::class)
```

Comment:
I had to change this scope in order to inject this classes in the AutofillService.
They are stateless and don't rely on the fragment context, so this should be non issue.